lib/removePipelineExtras.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+
+module.exports = removePipelineExtras;
+
+function removePipelineExtras(gltf) {

```

Comment:
I see - when you say check for extras, would I be able to do this if the nested glTF object didn't have an extras object, since they're optional? Would I need to separately keep track of the actual glTF properties hierarchy?

And just checking, is the removePipelineExtrasSpec file I added sufficient for unit testing (in this case)?